themes/bootstrap3/templates/search/results.phtml

```diff
@@ -57,29 +57,43 @@
     <? endif; ?>
     <?=$this->flashmessages()?>
     <div class="hidden-print search-controls row">
-      <div class="col-sm-6">
+      <div class="pull-left">
         <? if ($recordTotal > 0): ?>
-          <?=$this->transEsc("Showing")?>
-          <strong><?=$this->localizedNumber($this->results->getStartRecord())?></strong> - <strong><?=$this->localizedNumber($this->results->getEndRecord())?></strong>
+          <?
+            $transParams = [
+              '%%start%%' => $this->localizedNumber($this->results->getStartRecord()),
+              '%%end%%'   => $this->localizedNumber($this->results->getEndRecord()),
+              '%%total%%' => $this->localizedNumber($recordTotal),
+              '%%lookfor%%' => $this->escapeHtml($lookfor)
+            ];
+            $showingResults = '';
+          ?>
           <? if (!isset($this->skipTotalCount)): ?>
-            <? $this->layout()->srmessage = $this->transEsc('result_count', ['%%count%%' => $this->localizedNumber($recordTotal)]); ?>
-            <?=$this->transEsc('of')?> <strong><?=$this->localizedNumber($recordTotal)?></strong>
+            <? $showingResults = $this->translate('showing_results_of_html', $transParams); ?>
+          <? else: ?>
+            <? $showingResults = $this->translate('showing_results_html', $transParams); ?>
           <? endif; ?>
           <? if (isset($this->overrideSearchHeading)): ?>
+            <? echo $this->layout()->srmessage = $showingResults; ?>
             <?=$this->overrideSearchHeading?>
           <? elseif ($this->params->getSearchType() == 'basic'): ?>
-            <?=$this->transEsc('for search')?>: <strong>'<?=$this->escapeHtml($lookfor)?>'</strong>,
+            <? if (!isset($this->skipTotalCount)): ?>
+              <? $showingResults = $this->translate('showing_results_of_for_html', $transParams); ?>
+            <? else: ?>
+              <? $showingResults = $this->translate('showing_results_for_html', $transParams); ?>
+            <? endif; ?>
+            <? echo $this->layout()->srmessage = $showingResults; ?>
           <? endif; ?>
           <? if ($qtime = $this->results->getQuerySpeed()): ?>
-            <?=$this->transEsc('query time')?>: <?=$this->localizedNumber($qtime, 2).$this->transEsc('seconds_abbrev')?>
+            , <?=$this->transEsc('query time')?>: <?=$this->localizedNumber($qtime, 2).$this->transEsc('seconds_abbrev')?>
```

Comment:
Putting the comma here causes whitespace to appear in front of it, so we get:

Showing 1 - 20 results of 12,621 for search: 'test' , query time: 0.15s

when we actually want:

Showing 1 - 20 results of 12,621 for search: 'test', query time: 0.15s

Maybe I'm unusually bothered by free-floating commas, but I'd be happier if we could avoid the extra space.